### PR TITLE
Update Taquito and other dependencies in Tezos Coin Module

### PR DESCRIPTION
**Pull request by ac10n**
### ✅ Checklist

- [*] `npx changeset` was attached.
- [*] **Covered by automatic tests.** 
- [*] **Impact of the changes:** 
  - ...

### 📝 Description

_Replace this text by a clear and concise description of what this pull request is about and why it is needed. Be sure to explain the problem you're addressing and the solution you're proposing._
_For libraries, you can add a code sample of how to use it._
_For bugfixes, you can explain the previous behavior and how it was fixed._
_In case of visual features, please attach screenshots or video recordings to demonstrate the changes._

### ❓ Context

- **JIRA or GitHub link**: https://ledgerhq.atlassian.net/browse/LIVE-12866

---

### 🧐 Checklist for the PR Reviewers

- **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- **Any new dependencies** have been justified and documented.
- **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

### CODE REVIEW

File: pnpm-lock.yaml [hunk 1]
```diff
@@ -22973,13 +22973,30 @@ packages:
...
+    dependencies:
+      json-stringify-safe: 5.0.1
+    dev: false
+
   /@taquito/http-utils@13.0.1:
     resolution: {integrity: sha512-eHzd0HSL3qX6bOOSaQClm+0XmpbSNcJP69uzaBJwfXo7ntQR1bUfGLn6+1Hgsk/lJ0JxakD2PDA4aaeajHvyPw==}
     engines: {node: '>=6.0.0'}
     dependencies:
       axios: 0.26.1
     dev: false
```
[the remark this answers is not in the document]
Comment (ac10n) [reply pnpm-lock.yaml:22989]: Thank you. What about `bs58check`? I see it's used in other places as well, and I have upgraded its version in `coin-tezos` package but not anywhere else.

[the remark this answers is not in the document]
Comment (ac10n) [reply pnpm-lock.yaml]: Pushed a commit that fixes this inconsistency for `@taquito/ledger-signer`.

**Comment by jevonearth** [comment]
We conducted comparative testing between the current stable release of Ledger Live Wallet and the updated build in this PR. The testing is non-exhaustive and currently tests only against Tezos mainnet. Attached is a summary report of the functionality we did cover.
[Ledger Live Tezos Features Comparison with PR_ V3_wDisclaimer.pdf](https://github.com/user-attachments/files/15561172/Ledger.Live.Tezos.Features.Comparison.with.PR_.V3_wDisclaimer.pdf)

**Comment by jevonearth** [comment]
Latest testing results which now includes testing against the new "Paris" protocol on mainnet. 
[Ledger Live Tezos Features Comparison with PR - post Paris_ V4_wDisclaimer.pdf](https://github.com/user-attachments/files/15593922/Ledger.Live.Tezos.Features.Comparison.with.PR.-.post.Paris_.V4_wDisclaimer.pdf)

**Review by gre — APPROVED** [review]

**Review by KVNLS — APPROVED** [review]

**Comment by lvndry** [comment]
Closing PR as changes are handled here: https://github.com/LedgerHQ/ledger-live/pull/7041

**Pull request closed without merging**